client/webserver/api.go

```diff
@@ -242,6 +275,14 @@ func (s *WebServer) apiAccountImport(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	r.Close = true
+	if len(form.Pass) == 0 {
+		cachedPass, err := s.getCachedPassword(r)
+		if err != nil {
+			log.Errorf("unable to get cached pw: %v", err)
+		} else {
+			form.Pass = cachedPass
+		}
+	}
```

Comment:
This block is repeated enough to justify a helper method.

```Go
func (s *WebServer) resolvePass(appPW []byte) ([]byte, error) {
	if len(appPW) > 0 {
		return appPW, nil
	}
	cachedPass, err := s.getCachedPassword(r)
	if err != nil {
		if errors.Is(err, ErrNoCachedPW) {
			return nil, fmt.Errorf("app pass cannot be empty")
		}
		return nil, fmt.Errorf("error retrieving cached pw: %v", err)
	}
	return cachedPass, nil
}
```

where `ErrNoCachedPW` will be some constant error that identifies that there is no password cached.